packages/phishing-controller/src/utils.test.ts

```diff
@@ -0,0 +1,96 @@
+import sinon from 'sinon';
+import { applyDiffs, fetchTimeNow } from './utils';
+import { HotlistDiff } from './PhishingController';
+
+const exampleBlockedUrl = 'https://example-blocked-website.com';
+const exampleBlockedUrlOne = 'https://another-example-blocked-website.com';
+const exampleBlockedUrlTwo = 'https://final-example-blocked-website.com';
+const exampleBlocklist = [exampleBlockedUrl, exampleBlockedUrlOne];
+
+const exampleAllowUrl = 'https://example-allowlist-item.com';
+const exampleFuzzyUrl = 'https://example-fuzzylist-item.com';
+const exampleAllowlist = [exampleAllowUrl];
+const exampleFuzzylist = [exampleFuzzyUrl];
+const exampleListState = {
+  blocklist: exampleBlocklist,
+  fuzzylist: exampleFuzzylist,
+  tolerance: 2,
+  allowlist: exampleAllowlist,
+  version: 0,
+  name: 'MetaMask',
+  lastUpdated: 0,
+};
+
+const exampleAddDiff = {
+  targetList: 'blocklist',
+  url: exampleBlockedUrlTwo,
+  timestamp: 1000000000,
+} as HotlistDiff;
+
+const exampleRemoveDiff = {
+  targetList: 'blocklist',
+  url: exampleBlockedUrlTwo,
+  timestamp: 1000000000,
+  isRemoval: true,
+} as HotlistDiff;
```

Comment:
Nit: We should be able to get rid of these casts

```suggestion
const exampleAddDiff = {
  targetList: 'blocklist' as const,
  url: exampleBlockedUrlTwo,
  timestamp: 1000000000,
};

const exampleRemoveDiff = {
  targetList: 'blocklist' as const,
  url: exampleBlockedUrlTwo,
  timestamp: 1000000000,
  isRemoval: true,
};
```

This will leave an unused import (the `HotlistDiff` type) to get rid of as well